tests/ens/registrar/test_entries.py

```diff
@@ -0,0 +1,115 @@
+
+import pytest
+
+from web3.exceptions import StaleBlockchain
+
+from ens.registrar import Status
+
+
+def test_entries_passthrough(registrar, mocker, hash1, label1):
+    result = object()
+    mocker.patch.object(registrar, 'entries_by_hash', return_value=result)
+    mocker.patch.object(registrar.ens, 'labelhash', return_value=hash1)
+    assert registrar.entries(label1) is result
+    registrar.ens.labelhash.assert_called_once_with(label1)
+    registrar.entries_by_hash.assert_called_once_with(hash1)
+
+
+@pytest.mark.parametrize("dot", ['.', '．', '。', '｡'])
+def test_entries_use_label_with_dots(registrar, mocker, addr1, hash1, dot):
+    mocker.patch.object(registrar.ens, 'labelhash')
+    mocker.patch.object(registrar, 'entries_by_hash')
+    registrar.entries('holygrail%seth' % dot)
+    registrar.ens.labelhash.assert_called_once_with('holygrail')
+
+
+def test_entries_subdomain_meaningless(registrar, mocker, addr1, hash1):
+    mocker.patch.object(registrar.web3, 'sha3')
+    mocker.patch.object(registrar, 'entries_by_hash')
+    with pytest.raises(ValueError):
+        registrar.entries('montypythonandthe.holygrail.eth')
+
+
+def test_entries_status(registrar, mocker):
+    NUM_ENTRIES_STATUSES = 6
+    core_results = [[idx, None, 0, 0, 0] for idx in range(NUM_ENTRIES_STATUSES)]
+    mocker.patch.object(registrar.core, 'entries', side_effect=core_results)
```

Comment:
Can you explain what this test is actually testing?  Is there intermediate logic that's being tested?  If not, I think this is just testing the mock.